### PR TITLE
Add multiple email support

### PR DESCRIPTION
**Pull request by igregson**
Adds support for multiple emails. 

Required for website [PR 1068](https://github.com/azurestandard/website/pull/1068).

### CODE REVIEW

File: azure-providers.js [hunk 1]
```diff
@@ -273,10 +273,16 @@ var azureProvidersModule = angular
...
                         withCredentials: true,
                         headers: _headers,
-                    }
+                    };
+                    actions.get = {
+                        method: 'GET',
+                        url: url + '/email/:' + identifier,
+                        withCredentials: true,
+                        headers: _headers,
+                    };
```
Comment (wking) [inline azure-providers.js]: Isn't this handled by the [stock `get`][1]?

[1]: https://github.com/azurestandard/azure-angular-providers/blob/v0.8.5/azure-providers.js#L204-L208

Comment (igregson) [inline azure-providers.js]: Yes. Removed.

**Comment by igregson** [comment]
@wking Do I have the green light to merge this?

File: azure-providers.js [hunk 2]
```diff
@@ -49,6 +50,7 @@ var azureProvidersModule = angular
             'address': 'addresses',
             'category': 'categories',
             'country': 'countries',
+            'email': 'emails',
```
Comment (wking) [inline azure-providers.js]: You don't need this.  `_plurals` is just for irregular plurals.  Regular plurals (like “email” → “emails”) are handled [here][1].

[1]: https://github.com/azurestandard/azure-angular-providers/blob/v0.8.5/azure-providers.js#L170

Comment (igregson) [inline azure-providers.js]: Ahh... hadn't realized this. Makes sense.

**Pull request merged**